src/common/wallet/cointype.js

```diff
@@ -13,47 +14,60 @@ const coinType = {
     symbol: 'BTC',
   },
   RBTC: {
-    networkId: 137,
+    networkId: 30,
+    pathNetworkId: 137,
```

Comment:
```suggestion
    coinType: 137,
```

Let's give this the right name: `coinType`.

This matches the names used in the various standards documents:

- https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki
- https://github.com/satoshilabs/slips/blob/master/slip-0044.md
- https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP57.md

(applies to rest of this file too)

Reply:
I see. That makes sense. Okay we will do another sweep and rename the pathNetworkId to `coinType`. Yes other files need to be checked too.